libs/ngx-dsfr/search-bar/src/lib/search-bar.component.scss

```diff
@@ -8,6 +8,10 @@
   position: relative;
 }
 
+.input-wrapper {
+  margin-bottom: 0 !important;
+}
```

Comment:
I don't want `!important` in CSS files, it's considered a bad practice.

We can always do without it.

If the need is only for the `margin-bottom`, is it really necessary or can't we say that the developers using this Component will have to deal with that in their own layout ?